targets/UnrealMarketplacePlugin/makeBp.js

```diff
@@ -296,3 +298,29 @@ function getAuthBools(tabbing, apiCall) {
 
     return output;
 }
+
+function getCustomApiAssignmentLogic(tabbing, apiCall) {
+    if (apiCall.name === "ExecuteFunction")
+    {
+        return tabbing + "// Check for local debugging\n"
+            + tabbing + "FString localApiServer = PlayFabCommon::PlayFabCommonUtils::GetLocalSettingsFileProperty(TEXT(\"LocalApiServer\"));\n"
+            + tabbing + "if (!localApiServer.IsEmpty())\n"
+            + tabbing + "{\n"
+            + tabbing + "    FString endpoint = TEXT(\"" + apiCall.url + "\");\n"
+            + tabbing + "    endpoint.RemoveFromStart(TEXT(\"/\"));\n"
```

Comment:
This is a line that shouldn't be necessary if we're doing the url string construction consistently in all places.

Reply:
Resolving this as per our conversation in person.